lib/plugin-compat.php

```diff
@@ -21,14 +21,13 @@
  * plugins Jetpack, JP-Markdown, and WP Editor.MD
  *
  * @since 1.3.0
+ * @deprecated 5.0.0
  *
  * @param  array $post      Post object which contains content to check for block.
  * @return array $post      Post object.
  */
 function gutenberg_remove_wpcom_markdown_support( $post ) {
-	if ( class_exists( 'WPCom_Markdown' ) && has_blocks( $post['post_content'] ) ) {
-		WPCom_Markdown::get_instance()->unload_markdown_for_posts();
-	}
+	_deprecated_function( __FUNCTION__, '5.0.0' );
```

Comment:
Not sure what the version refers to here. Should it be the version where we remove it from Gutenberg? 5.1 or 5.2?

Reply:
Our use of `_deprecated_function` would probably be discouraged anyways, but as it's implemented, it's meant to serve to describe the version at which it became deprecated, not when it's to be removed.

>$version
>(string) (Required) The version of WordPress that deprecated the function.

https://developer.wordpress.org/reference/functions/_deprecated_function/